### DISCUSSION TITLE
Using Mods

### DISCUSSION THREAD

**Discussion by Laz55** [SheffeyG/CheatDeck#22] [General]
Hello

Let's say I  wanted to use mods along with cheatdeck  with Linux  you have to use some launch commands,but when your  using cheatdeck  it already has its own line of commands,so how do I combine  a new line of command  with the used with cheatdeck  for   if I used cheatdeck and use flings trainer for Horizon  Forbidden West  and I wanted to use this launch command  with a mod .

WINEDLLOVERRIDES="winhttp.dll=n,b  will it override  and not work with cheatddck?

**Comment by SheffeyG** [comment]
The overwrite problem has been fixed at #11

**Comment by Laz55** [comment]
Okay  thanks.
